pkg/handler/response.go

```diff
@@ -0,0 +1,11 @@
+package handler
+
+type Response struct {
```

Comment:
Shouldn't this be in the folder /internal/models?

Reply:
I thought about that, but my latter thought was that, since it's being used by the handlers alone. it might be okay if it resides in the handler. or what do you think?

Reply:
That is actually a good point.  
I might be shared between other handler implementations (since they also have to conform to the API), but I think this is in this case down to personal preference.

You don't need to change it.

Reply:
okay.